common/lib/common-utils/src/eventForwarder.ts

```diff
@@ -3,6 +3,7 @@
  * Licensed under the MIT License.
  */
 
+// eslint-disable-next-line import/no-nodejs-modules, unicorn/prefer-node-protocol
```

Comment:
I would like to enable these, but I ran into a lot of problems when I did. First TS started complaining about missing types, but when I added the node types, then it still didn't compile. I don't know how the compilation for this package is supposed to work since it has Node and browser implementations.

Reply:
The node packages get polyfilled automagically -- in the case of events, with https://www.npmjs.com/package/events

Actually, I see that in common-utils we even have `events` as an explicit dependency, though this should not be necessary.

Reply:
Note that while most of the time the polyfills work just fine, it's not unheard of that this can cause issues.  E.g. I had to dodge path/path-browserify differences here by explicitly using path-browserify:
https://github.com/microsoft/FluidFramework/blob/main/packages/dds/map/src/directory.ts#L40

Reply:
Looks like the `events `dependency was added as part of removing the node types: #2666.

But I find that really confusing. We clearly are importing some node built-ins -- why is `events `special but not, say, `fs` or `assert`?